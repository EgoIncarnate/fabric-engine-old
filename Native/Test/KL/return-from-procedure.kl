operator entry()
{
  return 17;
}
