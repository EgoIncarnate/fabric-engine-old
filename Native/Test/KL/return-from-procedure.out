3:3: error: functions with no return types do not return values
